app/javascript/components/mentoring/queue/SolutionList.jsx

```diff
@@ -31,9 +27,9 @@ export function SolutionList({ request, setPage }) {
 
   return (
     <div>
-      {isLoading && <Loading />}
-      {isError && <p>Something went wrong</p>}
-      {isSuccess && (
+      {status === 'loading' && <Loading />}
+      {status === 'error' && <p>Something went wrong</p>}
+      {status === 'success' && (
```

Comment:
What was the reason for this change? I sort of liked the previous version better (not that I mind it much)

Reply:
We pass in the status into this component now. If we pass each of the booleans, it will add extra props.

Reply:
Ah okay. Makes sense